src/components/components/appsco-list-item.html

```diff
@@ -59,6 +61,15 @@
             :host .item-message {
                 @apply(--appsco-list-item-message);
             }
+            appsco-account-image.preview-account-image {
+                margin-right: 8px;
+                --account-image: {
+                    width: 28px;
+                    height: 28px;
+                };
+                --account-initials-background-color: var(--group-role-initials-background-color);
```

Comment:
Why '--group-role-initials-background-color'?

Reply:
I've checked that. There  wasn't need for it. I've removed that style.